Data/models/Bookmark.swift

```diff
@@ -319,104 +265,85 @@ public class Bookmark: NSManagedObject, WebsitePresentable, Syncable {
         // If I save while the animation is happening, the rows look screwed up (draw on top of each other).
         // Adding a delay to let animation complete avoids this problem
         DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(250)) {
-            DataController.saveContext(context: frc?.managedObjectContext)
+            DataController.save(frc?.managedObjectContext)
         }
 
     }
 }
 
 // TODO: Document well
-// Getters
+// MARK: - Getters
 extension Bookmark {
-    fileprivate static func get(forUrl url: URL, countOnly: Bool = false, getFavorites: Bool = false, context: NSManagedObjectContext) -> AnyObject? {
-        let fetchRequest = NSFetchRequest<NSFetchRequestResult>()
-        fetchRequest.entity = Bookmark.entity(context: context)
+    fileprivate static func get(forUrl url: URL, countOnly: Bool = false, getFavorites: Bool = false, context: NSManagedObjectContext) -> [Bookmark]? {
+        let urlKeyPath = #keyPath(Bookmark.url)
+        let isFavoriteKeyPath = #keyPath(Bookmark.isFavorite)
+        
         let isFavoritePredicate = getFavorites ? "YES" : "NO"
```

Comment:
I know this was here before, but should be removed, and something like:
`getFavorites as NSNumber` || `NSNumber(value: getFavorites)` should be passed into the predicate instead.